### DISCUSSION TITLE
PPOCRV4识别模型训练时间、数据问题

### DISCUSSION THREAD

**Discussion by Wenqiang-Miao** [PaddlePaddle/PaddleOCR#12660] [Q&A]
请问方便透露PPOCRV4识别模型的训练数据规模吗？相比V3有变化么？训练进行了多少个epoch？能不能提供一下训练过程中的log，想看下训练过程中的损失下降情况，感谢。

**Comment by Wenqiang-Miao** [comment]
有人回复一下吗？

**Comment by SWHL** [comment]
这个估计只能参考官方放出来的技术报告了
